01.elements/mediapro/02.advanced/01.customizations/docs.md

```diff
@@ -4,4 +4,20 @@ taxonomy:
     category: docs
 ---
 
-[plugin:insert-content](/_partials/elements/customizations?mediapro)
+## Overrides
+
+Technically is possible to override any element file, but overriding the core files is not recommended as future updates could change the behavior and the override files became deprecated. However, sometimes is the only way to accomplish specific project requirements and if kept to minimal is a perfectly valid solution. Besides, by overriding and not directly editing the core files we assure our changes would remain after future extension updates.
+
+There are two paths for elements overriding, the global `media/zoo/custom_elements/mediapro` where the elements will be overridden always. And the `media/zoo/applications/{application}/elements/mediapro` where the override will apply only when the element is used by that specific App.
+
```

Comment:
What about `media/zoo/applications/{application}/templates/{template}/elements/mediapro/` override path?
Should it be documented also?